inc/file.class.php

```diff
@@ -198,24 +190,7 @@ private function getUploadedFile($uploadMandatory = false) {
     * @return string|boolean URL of the file
     */
    public function getFileURL() {
-      $config = Config::getConfigurationValues('flyvemdm', ['deploy_base_url']);
-      $deployBaseURL = $config['deploy_base_url'];
-      if ($deployBaseURL === null) {
-         return false;
-      }
-
-      $URL = $deployBaseURL . '/file/' . $this->fields['source'];
-      return $URL;
-   }
-
-   /**
-    * Create a directory
-    * @param string $dir
-    */
-   protected function createEntityDirectory($dir) {
-      if (!is_dir($dir)) {
-         @mkdir($dir, 0770, true);
-      }
+      return $this->getDeployableURL('file');
```

Comment:
Maybe we can replace calls to getFileURL() by getDeployableURL() directly (and do ther same in package).

This way, getDeployableURL() may be shared in the Deployable class.